File: javascript/solutions/different.js
// Problem: https://open.kattis.com/problems/different

let line;
const results = [];

while (line = readline()) {
  const numbers = line.split(' ');
  const firstNo = numbers[0];
  const secondNo = numbers[1];
  results.push(Math.abs(firstNo - secondNo));
}

print(results.join('\n'));

File: python3/solutions/different.py
# Problem: https://open.kattis.com/problems/different

########## GENERATE TEST CASES LOCALLY ##########

dummy_input = [
  '10 12',
  '71293781758123 72784',
  '1 12345677654321'
]

def input():
  return dummy_input.pop(0)

################## CODE #####################

import sys

for i in sys.stdin:
  ab = i.split()
  a = int(ab[0])
  b = int(ab[1])
  print(abs(a - b))